mlir/tools/mlir-miopen-lib/CMakeLists.txt

```diff
@@ -97,4 +90,56 @@ if( BUILD_FAT_LIBMLIRMIOPEN )
   install(TARGETS MLIRMIOpen
     ARCHIVE DESTINATION lib)
 
+  # Install Miir.h to ${CMAKE_INSTALL_PREFIX}/include/<project-name>-<version>/
+  # as part of component libMLIRMIOpen
+  set(PACKAGE_DIR MLIRMIOpen)
+  install(FILES Miir.h
+    DESTINATION include/${PACKAGE_DIR}
+    COMPONENT libMLIRMIOpen EXCLUDE_FROM_ALL)
+
+  # Install lib${target_name} as part of ${component_name} and export it to be searchable by find_package()
+  function(export_target target_name component_name)
+    set(export_set ${target_name}Targets)
+    set(pkg_config_file ${target_name}Config.cmake)
+    set(pkg_version_file ${target_name}ConfigVersion.cmake)
+
+    # Install lib${target_name}.a to ${CMAKE_INSTALL_PREFIX}/lib/<project-name>-<version>/
+    # and group it into the ${target_name}Targets export-set
+    install(TARGETS ${target_name}
+      EXPORT ${export_set}
+      ARCHIVE DESTINATION lib/${PACKAGE_DIR}
+      COMPONENT ${component_name} EXCLUDE_FROM_ALL)
+
+    # Generate package version file
+    include(CMakePackageConfigHelpers)
+    write_basic_package_version_file(
+      "${CMAKE_CURRENT_BINARY_DIR}/${pkg_version_file}"
+      VERSION ${CMAKE_PROJECT_VERSION}
```

Comment:
What version is it currently? I actually don't know if we have explicitly set this.

Reply:
It is set in the top-level CMakeLists.txt
`project(miopen-mlir VERSION 0.202108 LANGUAGES CXX C) `